app/routes/user.router.js

```diff
@@ -19,5 +19,6 @@ router.post('/updateProfile', jwtMiddleware.verify, authMiddleware.ensureLoggedI
 router.post('/changePassword', jwtMiddleware.verify, authMiddleware.ensureLoggedIn, userController.changePassword)
 router.get('/contributions', jwtMiddleware.verify, authMiddleware.ensureLoggedIn, userController.contributions)
 router.post('/updateBatch', jwtMiddleware.verify, authMiddleware.ensureOfficialPlacementTeam, userController.updateBatch)
+router.get('/achievements', jwtMiddleware.verify, authMiddleware.ensureLoggedIn, userController.achievements)
```

Comment:
We need to ensure that only student can access the achievements. It's any way there.

Reply:
ensureLoggedIn will enough for that I think

Reply:
Yeah, It will work but only students has achievements so **ensureStudent** would be more specific.

Reply:
But **spc** also have achievements and **ensureStudent** will not give permission to view achievements to SPCs.

Reply:
SPC will never have achievements. SPCs has two separate accounts one as normal student and another as SPC.